### DISCUSSION TITLE
Gradle

### DISCUSSION THREAD

**Discussion by EduUkenov** [cashapp/redwood#1836] [Q&A]
Is there an explanation somewhere of what to connect?

**Comment by JakeWharton** [comment]
There is no real documentation to speak of at this time. The best you'll get is to look at the samples. The counter sample showcases using Redwood for cross-platform UI development that's all compiled into the platform-specific binaries. The emoji search sample showcases using Treehouse for cross-platform UI where the UI bindings are compiled in, but the actual presenter which drives the UI is run inside a JS VM (although you still write it in Kotlin).

Someday we'll have more comprehensive docs, but it is not a priority at this time.

**Reply by EduUkenov** [reply]
Yes I understand. 
Thanks for the answer.
